### DISCUSSION TITLE
Infolist actions disabled instead of hidden

### DISCUSSION THREAD

**Discussion by ArtMin96** [filamentphp/filament#10932] [Help] [answered]
Infolist actions are disabled when I call hidden

**Comment by leandrocfe** [comment]
What is the issue? Share the code please

**Reply by ArtMin96** [reply]
I don't have that code now because I didn't find a solution and solved the problem in another way. I called `registerActions` with Infolist and set `Action` to hidden, but instead of disappearing, it was disabled.

**Reply by leandrocfe** [reply, marked as answer]
Yes, because you need to check the visibility in the view

https://discord.com/channels/883083792112300104/1184064527298478111/1184894000784740393

**Reply by ArtMin96** [reply]
Thanks. I thought it should work with hidden.
